staging/kubefed/Chart.yaml

```diff
@@ -1,8 +1,9 @@
 apiVersion: v1
 description: KubeFed helm chart
 name: kubefed
-version: 0.0.3
+version: 0.0.4
 appVersion: v0.1.0-rc6
```

Comment:
just a side note, there's a more recent release of this. Might wanna drop a JIRA reminder to update later.

Reply:
I need to discuss that with yackl folks, cannot take that decision, also they have a chart that needs work... a whole other topic.